src/mlpack/methods/logistic_regression/logistic_regression_main.cpp

```diff
@@ -327,6 +334,22 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
     }
   }
 
+  // Did we want training accuracy?
+    if (params.Has("print_training_accuracy"))
+    {
+      timers.Start("lr_prediction");
+      arma::Row<size_t> predictions;
+      model->Classify(regressors, predictions);
+
+      const size_t correct = arma::accu(predictions == responses);
+
+      Log::Info << correct << " of " << responses.n_elem << " correct on training"
+          << " set (" << (double(correct) / double(responses.n_elem) * 100) << ")."
+          << endl;
+      timers.Stop("lr_prediction");
+    }
+  }
```

Comment:
It seems like there's an extra brace here, and things are indented more than they need to be.